pkg/controller/networkpolicy/networkpolicy_controller_test.go

```diff
@@ -3271,6 +3274,59 @@ func TestGetAddressGroupMemberSet(t *testing.T) {
 	}
 }
 
+func TestAddNodeSelector(t *testing.T) {
```

Comment:
There is neither a function called AddNodeSelector nor a scenario that "add node selector".
I think it should be called "TestAddressGroupWithNodeSelector"

Reply:
Done.